dashboard/test/lib/services/user/gender_normalizer_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+
+class Services::User::GenderNormalizerTest < ActiveSupport::TestCase
+  let(:described_instance) {described_class.new(raw_input: raw_input)}
+  let(:raw_input) {nil}
+
+  describe '#call' do
+    subject(:normalize) {described_instance.call}
+
+    context 'when female' do
+      f_values = %w(f F female Female girl gal woman feminine she her mujer femail fem 여자 여 여성 kız kadın perempuan أنثى بنت انثي žena weiblich vrouw meisje kobieta fille féminin donna chica kvinde pige tjej kvinna ស្រី famale gril)
+      f_values.each do |value|
+        context "with input #{value}" do
+          let(:raw_input) {value}
+
+          it "returns #{Services::User::GenderNormalizer::FEMALE}" do
+            _(normalize).must_equal Services::User::GenderNormalizer::FEMALE
```

Comment:
The `subject` wrapper creates an expect-like method for the defined subject:
```suggestion
            _normalize.must_equal Services::User::GenderNormalizer::FEMALE
```